app/scripts/modules/core/src/pipeline/status/ArtifactList.spec.tsx

```diff
@@ -15,7 +15,7 @@ describe('<ArtifactList/>', () => {
   beforeEach(mock.inject(() => {})); // Angular is lazy.
 
   it('renders null when null artifacts are passed in', function() {
-    const artifacts: IArtifact[] = [];
+    const artifacts: IArtifact[] = null;
```

Comment:
doh.  I should've written an ArtifactList.spec.tsx.spec.tsx.

Reply:
Test Driven Testing